src/models/vector_db_metadata.py

```diff
@@ -10,19 +11,22 @@ class VectorDBMetadata(Base):
     vector_db_type = Column(Enum(VectorDBType))
     index_name = Column(String)
     environment = Column(String)
+    collection = Column(String)
 
     def serialize(self):
         return {
             'vector_db_type': self.vector_db_type.name if self.vector_db_type else None,
             'index_name': self.index_name,
             'environment': self.environment,
+            'collection': self.collection
         }
     
     @staticmethod
     def _from_request(request):
-        vector_db_metadata_dict = json.loads(request.form.get('VectorDBMetadata'))
+        vector_db_metadata_dict = defaultdict(lambda : "", json.loads(request.form.get('VectorDBMetadata')))
         vector_db_metadata = VectorDBMetadata(
             vector_db_type = VectorDBType[vector_db_metadata_dict['vector_db_type']], 
             index_name = vector_db_metadata_dict['index_name'], 
-            environment = vector_db_metadata_dict['environment'])
+            environment = vector_db_metadata_dict['environment'],
+            collection = vector_db_metadata_dict['collection'])
```

Comment:
Can we make this optional somehow?

Reply:
Maybe have a default value if 'collection' is not in the dict

Reply:
That's why I was using the [defaultdict](https://docs.python.org/3/library/collections.html#collections.defaultdict), I can use a different logic.